lib/transport/selenium-webdriver/method-mappings.js

```diff
@@ -582,10 +582,13 @@ module.exports = class MethodMappings {
 
           return value;
         },
-
+        
         async clearElementValue(webElementOrId) {
           const element = this.getWebElement(webElementOrId);
           await element.clear();
+          var value = await element.getAttribute("value");
+          const backArr = Array(value.length).fill(Key.BACK_SPACE);
+          await element.sendKeys(...backArr);
```

Comment:
@FireNdIce3 Always create a new branch (with the `main` branch as base) while starting to work on a new issue.

Reply:
sorry mb
I accidentally mixed both of them
Apologies